crates/aptos/src/account/create.rs

```diff
@@ -45,47 +50,33 @@ impl CreateAccount {
         let address = account_address_from_public_key(&public_key_to_create);
 
         if self.use_faucet {
-            self.create_account_with_faucet(address).await
+            let faucet_url = if let Some(faucet_url) = self.faucet_url {
+                faucet_url
+            } else if let Some(url) = CliConfig::load()?.faucet_url {
+                Url::parse(&url)
+                    .map_err(|err| CliError::UnableToParse("config faucet_url", err.to_string()))?
+            } else {
+                Url::parse(DEFAULT_FAUCET_URL).map_err(|err| {
+                    CliError::UnexpectedError(format!("Failed to parse default faucet URL {}", err))
```

Comment:
Fine since it's not used anywhere else for now, but should be moved into a shared function of some sort.

Reply:
Yeah, I figured it would be a shared type if that occurs again